cmd/serve/grant.go

```diff
@@ -1,16 +1,20 @@
-package main
+package serve
 
 import (
 	"context"
 	"fmt"
 	"net/http"
-	_ "net/http/pprof"
+
+	// _ "net/http/pprof"
 	"os"
 	"time"
 
 	"github.com/asaskevich/govalidator"
+	"github.com/spf13/cobra"
+
 	// re-using viper bind-env for wallet env variables
-	_ "github.com/brave-intl/bat-go/cmd/wallets"
+	"github.com/brave-intl/bat-go/cmd"
+	// _ "github.com/brave-intl/bat-go/cmd/wallets"
```

Comment:
This is needed to include the wallets.init() startup which allows for wallets env variable bindings.  We can remove this when we completely disassociate running wallets in grants containers.

Reply:
re-orged the imports: 93b2085

Reply:
:+1: